src/store/modules/user/mutations.ts

```diff
@@ -16,6 +16,9 @@ const mutations: MutationTree <UserState> = {
     },
     [types.USER_CURRENT_FACILITY_UPDATED] (state, payload) {
         state.currentFacility = payload;
+    },
+    [types.USER_INSTANCE_URL_UPDATED] (STATE, payload) {
+        STATE.instanceUrl = payload;
```

Comment:
Change `STATE` to `state`

Reply:
Changed.
Thank You for reviewing my code.